<?xml version="1.0"?>

<launch>

    <!--  ****** MPC Node ******  -->
    <rosparam file="$(find nonlinear_mpc_casadi)/params/mpc_params.yaml" command="load" />
    <node name="mpc_node" pkg="nonlinear_mpc_casadi" type="Nonlinear_MPC.py" output="screen" >
        <param name="waypoints_filepath" value="$(find nonlinear_mpc_casadi)/waypoints/kelley_third_floor.csv"/>
    </node>

</launch>
